pkgs/development/libraries/cosmopolitan/default.nix

```diff
@@ -0,0 +1,68 @@
+{ lib, gcc9Stdenv, fetchFromGitHub, runCommand, cosmopolitan }:
+
+gcc9Stdenv.mkDerivation {
+  pname = "cosmopolitan-unstable";
+  version = "20210228";
+  src = fetchFromGitHub {
+    owner = "jart";
+    repo = "cosmopolitan";
+    rev = "94afa982c35d4322999b5591f6d41d5cfc3f19a5";
+    sha256 = "sha256-LaXkFMoArg5LUw4AdyhOqxzSJLdQCwHVBK2+lXVmxuI=";
+  };
+
+  postPatch = ''
+    patchShebangs build/
+    rm -r third_party/gcc
+  '';
+
+  dontConfigure = true;
+  dontFixup = true;
+  enableParallelBuilding = true;
+
+  preBuild = ''
+    makeFlagsArray=(SHELL=/bin/sh AS=as CC=gcc GCC=gcc CXX=g++ LD=ld OBJCOPY=objcopy "MKDIR=mkdir -p")
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/{bin,lib/include}
+    install o/cosmopolitan.h $out/lib/include
+    install o/cosmopolitan.a o/libc/crt/crt.o o/ape/ape.{o,lds} $out/lib
+    cat > $out/bin/cosmoc <<EOF
+    #!$SHELL
+    exec ${gcc9Stdenv.cc}/bin/gcc \
```

Comment:
```suggestion
    exec ${gcc9Stdenv.cc.targetPrefix}gcc \
```

Reply:
Not quite working as expected. This is the empty string during normal builds, then gcc is no longer in PATH.

Reply:
i'm not the strongest on knowing which things will be needed for cross-compilation, pushing my latest, but needs more review